### PR TITLE
Pull PDFs

### PR DESCRIPTION
**Pull request by KatiRG**
## What this PR accomplishes
Searches for files in a data drive that match the file names given in a lookup table; copies the files to a destination directory and outputs a results CSV file that records the status ("found", "missing") of each requested file.

Built by making small modifications to `file_extractor/pull_collision_images.py`. 

## Issue addressed
#27 

## What needs review
- is the Readme clear enough?
- any comments on the script?

### CODE REVIEW

**Comment by radumas** [comment]
I think the pdf and collision puller should be in the same folder and some similarities in their instructions can be combined into one README. Worth checking if the `tif` files have the same naming convention as the PDFs.

my preference for the new pdf code would be to try to organize so that we have 1 script with a few options specified by some all caps `VARIABLE` at the top. e.g.

```python
FILE_FORMAT = 'PDF' #'TIF'
YYMM_FOLDER_STRUCTURE = FALSE
```

and then pass those to the functions.

i guess it's an open question which file format we will be requested to pull in the future.

**Comment by jwrcoleman** [comment]
I agree with this. We could conceivably be asked for TIF from one consultant and PDF from another.  Can we make the output folder in //trafic instead of K drive? (or just have this a parameter). This should for sure be one tool (collision puller and pdf puller)

**Comment by KatiRG** [comment]
> 
> 
> I agree with this. We could conceivably be asked for TIF from one consultant and PDF from another. Can we make the output folder in //trafic instead of K drive? (or just have this a parameter). This should for sure be one tool (collision puller and pdf puller)

The user can define whatever destination drive they want. What would you like for the default?

**Comment by radumas** [comment]
Jesse suggested
```
\\trafic\Documents\TDCSB\PROJECT\REQUESTS\_CONTRACTED_SERVICES\Work Packages\Image_Puller
```

**Comment by aharpalaniTO** [comment]
My general feedback after using the puller:
- works great! super easy to use
- some suggestions on documentation:
1. clean up or split the second step under the "The script goes through the following steps" in the README. As written, it's pretty hard to follow.
2. provide a small sample (either by including a CSV template with examples or a screenshot in the readme) of what the input file should look like
3. I believe PDFs only exist for collisions after a certain year - add that disclaimer to the documentation

**Review by aharpalaniTO — APPROVED** [review]
Thanks @KatiRG - looks great.
